client/src/components/GlobalStyles.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Global, css } from '@emotion/core';
+
+function GlobalStyles() {
+  return (
+    <Global
+      styles={css`
+        body {
```

Comment:
don't forget to declare `box-sizing: border-box;` for all elements as well `*, *::before, *::after` ;)